packages/ui/package.json

```
@@ -6,9 +6,11 @@
     "node": ">= 16"
   },
   "scripts": {
-    "dev": "yarn build:nexus && next dev",
-    "build": "yarn build:nexus && next build",
-    "build:nexus": "ts-node-dev -O '{ \"module\": \"commonjs\" }' --transpile-only graphql/schema.ts && prettier --write graphql/generated/*",
+    "dev": "yarn build:nexus && yarn build:graphql && next dev",
+    "build": "yarn build:nexus && yarn build:graphql && next build",
```

These are doing extra work

```suggestion
    "dev": "yarn build:graphql && next dev",
    "build": "yarn build:graphql && next build",
```